apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -2209,6 +2209,7 @@ nameservice_transactions(_Config) ->
     nameservice_transaction_update(MinerAddress, MinerPubkey),
     nameservice_transaction_transfer(MinerAddress, MinerPubkey),
     nameservice_transaction_revoke(MinerAddress, MinerPubkey),
+    nameservice_transaction_subname(MinerAddress, MinerPubkey),
```

Comment:
Please update comment at top or delete it.

Reply:
updated comment